ASM/src/config.asm

```diff
@@ -242,6 +242,9 @@ CFG_BIGOCTO_OVERRIDE_KEY:
 .area 6, 0x00
 PASSWORD:
 .endarea
+.area 14, 0x00
+CFG_DUNGEON_PRECOMPLETED:
+.endarea
```

Comment:
Is this aligned such that it won't cause issues on N64?

Reply:
I'll try to do further tests on Ares tomorrow to be safe

Reply:
It's used in the C code as an array of bytes, so that shouldn't cause any alignment issues, right?

Reply:
Double checked in Ghidra and you're right. the values are loaded via `lb` so there shouldn't be any issues.